### PR TITLE
build(combinator): DLT-2046 DLT-2047 update bundler and dependencies

### PR DESCRIPTION
**Pull request by juliodialpad**
# Update combinator bundler and dependencies

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] Build system

## :book: Jira Ticket

- https://dialpad.atlassian.net/browse/DLT-2046
- https://dialpad.atlassian.net/browse/DLT-2047

## :book: Description

- Migrated bundler from vue-cli to vite.
- Removed few unneeded files
- Couple visual fixes
- Commented `getIcons` function until I figure out exactly what was it used for.
- Migrated old dialtone variables.
- Updated icons
- Didn't found a direct mapping on some icons that were used, changed for the icon I thought seem similar or correct. We can review and update this later with Design.

## :bulb: Context

Combinator package was using vue-cli and we want to standardize vite across our packages.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [ ] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

## :crystal_ball: Next Steps

- Migrate tests to vitest

### CODE REVIEW

**Comment by braddialpad** [comment]
Is this ready for review? Looks like we're still having some dependency issues.

**Comment by juliodialpad** [comment]
> Is this ready for review? Looks like we're still having some dependency issues.

Forgot to update pnpm-lock file, this should be ready for review.

File: packages/combinator/src/assets/themes/light.less [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+  --dtc-theme-color-foreground-lighten: var(--dt-color-black-700);
 
-  --dtc-theme-color-background: var(--white);
+  --dtc-theme-color-background: var(--dt-color-neutral-white);
   --dtc-theme-color-background-darken: var(--black-050);
-  --dtc-theme-color-background-lighten: var(--white);
+  --dtc-theme-color-background-lighten: var(--dt-color-neutral-white);
 
   color-scheme: light;
 }
```
Comment (braddialpad) [inline packages/combinator/src/assets/themes/light.less:11]: This of course needs to be updated to use proper dialtone theming now. Not a big priority though.

**Review by braddialpad — APPROVED** [review]
A bit hard to review since I can't actually run the combinator yet, but I went through all the code, nothing that stood out to me as incorrect.

File: packages/combinator/src/assets/themes/light.less (hunk 1 above)
Comment (juliodialpad) [reply packages/combinator/src/assets/themes/light.less:11]: Yes, I will defer that for a later stage when Design gives us something to start working with.

**Pull request merged**